app/modules/visualization/js/objectBarsMultiple.js

```diff
@@ -256,11 +481,31 @@ function wrap(text, width) {
         	//var range = 10;
         	return range;
         })
+		.attr("class", function (d, i) {
+			
+			var find = d.Key;
+			/*
+			var res = find.split("_");
+			var re = '';
+			if (res.length>0) {
+				re = res[res.length-1];
+			}				
+			*/
+			var re = find.replace(/ /g, '_');
+			re = re.replace(/\'/g, '_');
+			re = re.replace(/"/g, '_');
+			re = re.replace(/,/g, '_');
+			re = re.replace(/\[/g, '_');
+			re = re.replace(/\]/g, '_');

```

Comment:
also note that the max Unicode is U+10FFFF but it looks like you can only use up to U+FFFF within the regex, which basically covers *all* possible characters used in Policycompass..